.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,81 @@
+name: Build and Push
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  buildx:
+    name: Build and Push the Docker Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compute Tag
+        uses: actions/github-script@v6
+        id: compute-tag
+        with:
+          result-encoding: string
+          script: |
+            if (context.eventName === "pull_request") {
+              return "pr-" + context.issue.number;
+            }
+            if (context.eventName === "push") {
+              if (context.ref.startsWith("refs/tags/")) {
+                return context.ref.slice(10);
+              }
+              if (context.ref === "refs/heads/develop") {
+                return "latest";
+              }
+            }
+            return "FALSE";
+
+      - uses: actions/checkout@v2
+
+      -
+        name: Docker Login
+        id: docker-login
+        env:
+          DOCKER_USER: ${{secrets.DOCKER_USERNAME}}
+          DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+        run: |
+          docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
```

Comment:
You can use `${{secrets.DOCKER_USERNAME}}` and `${{secrets.DOCKER_PASSWORD}}` directly here